lib/mix/tasks/gen/ash.gen.resource.ex

```diff
@@ -371,10 +381,19 @@ defmodule Mix.Tasks.Ash.Gen.Resource do
   end
 
   defp resolve_type(value) do
-    if String.contains?(value, ".") do
-      Module.concat([value])
-    else
-      String.to_atom(value)
-    end
+    resolved_type =
```

Comment:
Not really happy with how this reads, but saw a recent tweet about using the simplest abstractions and that's my attempt.